### ISSUE TITLE
No chapters cyborg-tl.com

### ISSUE THREAD

**Issue by pavan3999** [dteviot/WebToEpub#1336]
**Describe the bug**
when opening any novel from cyborg-tl.com through webtoepub extension no chapters showing

**To Reproduce**
1. Go to '[cyborg-tl.com](https://cyborg-tl.com/series/starting-by-getting-a-wife-it-turns-out-she-is-a-skilled-assassin/)'
2. Click on 'webtoepub extension'
3. shows no chapters and cover image

**Expected behavior**
shows number of chapters and able to download epub chapters including

**Screenshots**
![Screenshot_2024-06-06-14-16-15-188_com kiwibrowser browser](https://github.com/dteviot/WebToEpub/assets/23577468/2a4c61d7-c136-4126-910e-2b7861d05d81)

**Smartphone (please complete the following information):**
 - Device: Poco M6 Pro 5g
 - OS: Android 14
 - Browser: Kiwibrowser
 - Version: 124.0.6327.4

**Comment by Kiradien** [comment]
Pull request added with a fix; just had to add to an existing parser.

**Comment by dteviot** [comment]
@Kiradien 
There already was a parser for the old site layout.  I needed to remove it to make your changes work.

@pavan3999 

Test versions for Firefox and Chrome have been uploaded to https://drive.google.com/drive/folders/1B_X2WcsaI_eg9yA-5bHJb8VeTZGKExl8?usp=sharing. Pick the one suitable for you, follow the "How to install from Source (for people who are not developers)" instructions at https://github.com/dteviot/WebToEpub/tree/ExperimentalTabMode#user-content-how-to-install-from-source-for-people-who-are-not-developers and let me know how it goes.
Tested with:
* https://cyborg-tl.com/series/starting-by-getting-a-wife-it-turns-out-she-is-a-skilled-assassin/, chapter 1

For my notes: 18 minutes work
